src/app/data_harvester/temperature/heim.rs

```diff
@@ -89,7 +89,51 @@ pub async fn get_temperature_data(
             let temp_label = path.join(name.replace("input", "label"));
             let temp_label = fs::read_to_string(temp_label).ok();
 
-            let name = match (&hwmon_name, &temp_label) {
+            // Do some messing around to get a more sensible name for sensors
+            //
+            // - For GPUs, this will use the kernel device name, ex `card0`
```

Comment:
I'm wondering if it would be good to still keep the GPU name (e.g. `amdgpu`) as part of the name or label, since that might still be useful information for a user that we're throwing away.

Reply:
Addressed in latest commit

Now displays as, for example, `card1 (amdgpu): edge`.